### PR TITLE
jira workitems

### PR DESCRIPTION
**Pull request by tsurdilo**
workitems to work with Jira

### CODE REVIEW

File: jira-workitem/pom.xml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.powermock</groupId>
+      <artifactId>powermock-module-junit4</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-test</artifactId>
```
Comment (mswiderski) [inline jira-workitem/pom.xml]: this should be in scope test

File: jira-workitem/pom.xml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    </dependency>
+    <dependency>
+      <groupId>org.jbpm</groupId>
+      <artifactId>jbpm-test-util</artifactId>
```
Comment (mswiderski) [inline jira-workitem/pom.xml]: same here, scope test

File: jira-workitem/src/main/java/org/jbpm/process/workitem/jira/AddCommentOnIssueWorkitemHandler.java [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                                                                      commentVisibleTo),
+                                                       null);
+                    auth.getIssueRestClient().addComment(progressMonitor,
+                                                         issue.getSelf(),
+                                                         toAddComment);
+
+                    workItemManager.completeWorkItem(workItem.getId(),
+                                                     null);
+                } else {
+                    logger.warn("Could not find issue with key: " + issueKey);
```
Comment (mswiderski) [inline jira-workitem/src/main/java/org/jbpm/process/workitem/jira/AddCommentOnIssueWorkitemHandler.java]: shouldn't this throw an exception?

File: jira-workitem/src/main/java/org/jbpm/process/workitem/jira/ResolveIssueWorkitemHandler.java [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+                                                                          fieldInputs,
+                                                                          Comment.valueOf(resolutionComment));
+                    auth.getIssueRestClient().transition(issue.getTransitionsUri(),
+                                                         transitionInput,
+                                                         progressMonitor);
+
+                    workItemManager.completeWorkItem(workItem.getId(),
+                                                     null);
+                } else {
+                    logger.warn("Could not find issue with key: " + issueKey);
```
Comment (mswiderski) [inline jira-workitem/src/main/java/org/jbpm/process/workitem/jira/ResolveIssueWorkitemHandler.java]: same here, shouldn't this throw an exception?

**Comment by tsurdilo** [comment]
@mswiderski updated - thanks!! the test scope issue was also in other workitem modules so i fixed it there too.

**Comment by tsurdilo** [comment]
@mswiderski this ok to merge? thanks

**Pull request merged**